op-node/rollup/driver/state.go

```diff
@@ -482,6 +399,14 @@ func (s *Driver) ResetDerivationPipeline(ctx context.Context) error {
 	}
 }
 
+func (s *Driver) OnUnsafeL2Payload(ctx context.Context, payload *eth.ExecutionPayloadEnvelope) error {
+	s.emitter.Emit(p2p.ReceivedBlockEvent{
+		From:     "",
```

Comment:
What's the `From` field used for? Does it make sense to encode here that this payload comes from the admin api?